runtime/tools/spec-builder/test/build-spec.spec.js

```diff
@@ -0,0 +1,61 @@
+const assert = require('assert');
+const fs = require('fs');
+const Utils= require('./utils/spawn-child-process');
+
+describe('Build spec tests', () => {
+    before(async function() {
+        await Utils.emptyActualRawFiles();
+    });
+
+    describe('Build spec generates correct raw outputs', function() {
```

Comment:
There is no point in having this nested `describe`. Usually nested describes are used when you have another set of `it` tests that want to share/use the same `before` / `beforeEach`/ `after` or `afterEach` logic. Otherwise, they are of no purpose.